com.zandronum.Zandronum.appdata.xml

```diff
@@ -51,8 +51,8 @@
       <caption>Doom cooperative</caption>
     </screenshot>
     <screenshot>
-      <image>https://i.imgur.com/u5Wb5pw.png</image>
-      <caption>ZDoom Wars competitive</caption>
+      <image>https://i.imgur.com/Qp7kJzY.png</image>
```

Comment:
I would prefer if the screenshots are hosted on this repository.

Reply:
Smooth timing. 

I'll change it, but this should have been a bit more clear from the outset. Many other packages also load their images from third party image hosts.

Could you add this to the wiki?
https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
We prefer if the screenshots are hosted by the upstream developer. Hosting screenshots on the same repository as the appdata will probably become a strict rule with time.
Do you mind adding that to the wiki? Otherwise I will look into it. Thanks

Reply:
I don't have access to it, but if you can paste this at line 100, that should do it

```
### Images

A screenshot presents your application to the outside world, and could be seen by many people. They should be in a 16:9 ratio, and they should be representative of the application. See all screenshot requirements at the offical documentation.

For including screenshots in regards to Flathub, two options are possible:

 * They are part of the upstream project, and you link to them directly
 * You made them for Flathub. In that case, you are required to upload screenshots to Github
```